woodwork/tests/schema/test_schema.py

```diff
@@ -162,3 +167,93 @@ def test_schema_column_metadata(sample_column_names, sample_inferred_logical_typ
 
     schema = Schema(sample_column_names, sample_inferred_logical_types, column_metadata={'id': column_metadata})
     assert schema.columns['id']['metadata'] == column_metadata
+
+
+def test_filter_schema_cols(sample_column_names, sample_inferred_logical_types):
+    schema = Schema(sample_column_names, sample_inferred_logical_types,
+                    time_index='signup_date',
+                    index='id',
+                    name='dt_name')
+
+    filtered = schema._filter_cols(include=Datetime)
+    assert filtered == ['signup_date']
+
+    filtered_log_type_string = schema._filter_cols(include='NaturalLanguage')
+    filtered_log_type = schema._filter_cols(include=NaturalLanguage)
+    assert filtered_log_type == filtered_log_type_string
+
+    filtered_semantic_tag = schema._filter_cols(include='numeric')
+    assert filtered_semantic_tag == ['age']
+
+    filtered_multiple = schema._filter_cols(include=['numeric'])
+    expected = ['phone_number', 'age']
+    for col in filtered_multiple:
+        assert col in expected
```

Comment:
Could also make expected a set and use `assert set(filtered_multiple) == expected` instead of looping.

Reply:
Another problem with this test: `phone_number` isn't even in `filtered_multiple`, so I'm not sure what this is even testing, since this call to `_filter_cols` doesn't even produce multiple columns.

Think I'm going to add a line where we check that `schema._filter_cols(include=NaturalLanguage)` matches an expected set since that will produce multiple columns.